docs/usage.md

```diff
@@ -55,11 +55,57 @@ spec:
 Once this is created the _external-dns-operator_ will create a deployment of _external-dns_ which is configured to
 manage DNS records in AWS Route53.
 
-## AWS GovCloud
+## AWS Assume Role
+
+The _external-dns-operator_ supports managing records in another AWS account's hosted zone. To achieve this, you will
+require an IAM Role ARN with the necessary permissions properly set up. This Role ARN should then be specified in the
+`ExternalDNS` resource in the following manner:
+
+```yaml
+apiVersion: externaldns.olm.openshift.io/v1beta1
+kind: ExternalDNS
+metadata:
+  name: aws-example
+spec:
+  provider:
+    type: AWS
+    aws:
+      credentials:
+        name: aws-access-key
+      assumeRole:
+        arn: arn:aws:iam::123456789012:role/role-name # Replace with the desire Role ARN
+  zones: # Replace with the desired hosted zone IDs
+    - "Z3URY6TWQ91KXX"
+  source:
+    type: Service
+    fqdnTemplate:
+    - '{{.Name}}.mydomain.net'
+```
+
+**Note**: Due to a limitation of the `v1beta1` API requiring the `credentials` field, OpenShift users will be required
```

Comment:
BTW I am planning to add a section on using a Shared VPC to OpenShift's docs: https://docs.openshift.com/container-platform/4.13/networking/external_dns_operator/nw-creating-dns-records-on-aws.html

If you think we don't need both (this repo docs and OpenShift docs), I can drop this one. But I figure better safe than sorry.

I know the docs in this repo are not just for OpenShift users, but it does feel a little strange there isn't a mention of the Cloud Credential Operator providing the credentials for OpenShift anywhere. I wonder if we should introduce that concept somewhere around here, as a new user, that wasn't clear to me I didn't need credentials from reading these docs.

I feel like we should also have a `aws-openshift.md` doc just like the azure and gcp openshift docs in https://github.com/openshift/external-dns-operator/blob/main/docs. That's seems odd to me.

Reply:
Okay thanks. I'll leave what I have now, and possibly open another PR later with details on `CredentialsRequest` to keep this PR cleaner.